packages/core/src/collision/b2_broad_phase.ts

```diff
@@ -215,8 +215,11 @@ export class b2BroadPhase<T> {
         ++this.m_moveCount;
     }
 
-    private UnBufferMove(proxy: b2TreeNode<T>): void {
-        const i = this.m_moveBuffer.indexOf(proxy);
-        this.m_moveBuffer[i] = null;
-    }
+    public UnBufferMove(proxy: b2TreeNode<T>): void {
+        for (let i: number = 0; i < this.m_moveCount; ++i) {
```

Comment:
The type information is redundant here. Please remove it.

Reply:
Are you talking about `: number`? It's now removed.